crates/bevy_render/src/texture/image.rs

```diff
@@ -131,7 +131,11 @@ impl Default for Image {
                 sample_count: 1,
                 usage: wgpu::TextureUsages::TEXTURE_BINDING | wgpu::TextureUsages::COPY_DST,
             },
-            sampler_descriptor: wgpu::SamplerDescriptor::default(),
+            sampler_descriptor: wgpu::SamplerDescriptor {
+                mag_filter: wgpu::FilterMode::Linear,
```

Comment:
These should have comments explaining why we've diverged from `wgpu`'s defaults

Probably link to the original issue thread as well.